aws/security_group/main.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_security_group" "this" {
+  name        = var.name
+  description = var.description
+  vpc_id      = var.vpc_id
+
+  dynamic "ingress" {
+    for_each = var.ingress_rules
+    content {
+      description = ingress.value.description
+      from_port   = ingress.value.from_port
+      to_port     = ingress.value.to_port
+      protocol    = ingress.value.protocol
+      cidr_blocks = ingress.value.cidr_blocks
+    }
+  }
+
+  egress {
```

Comment:
I wouldn't leave this one hardcoded, as there are cases we don't need it like on RDSs, Redis caches, etc.
Better if people pass these egress rules to the module just like the ingress ones